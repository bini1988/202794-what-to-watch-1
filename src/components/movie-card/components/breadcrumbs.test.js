import React from "react";
import {MemoryRouter} from "react-router-dom";
import renderer from "react-test-renderer";
import Breadcrumbs from "./breadcrumbs";
import card from "../../../mocks/movie-card";

it(`Breadcrumbs correctly renders default markup`, () => {
  const tree = renderer.create(
      <MemoryRouter
        initialEntries = {[`/`]}>
        <Breadcrumbs card={card}/>
      </MemoryRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
