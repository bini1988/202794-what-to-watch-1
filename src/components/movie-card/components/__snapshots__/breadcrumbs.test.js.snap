// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs correctly renders default markup 1`] = `
<nav
  className="breadcrumbs"
>
  <ul
    className="breadcrumbs__list"
  >
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
        href="/film/1"
        onClick={[Function]}
      >
        War of the Worlds
      </a>
    </li>
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
        href="/film/1/review"
        onClick={[Function]}
      >
        Add review
      </a>
    </li>
  </ul>
</nav>
`;
